src/test/java/it/gov/pagopa/payment/service/qrcode/QRCodePreAuthServiceImplTest.java

```diff
@@ -25,20 +25,23 @@
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.http.HttpStatus;
 
+import java.util.List;
+
 @ExtendWith(MockitoExtension.class)
 class QRCodePreAuthServiceImplTest {
 
   @Mock private TransactionInProgressRepository transactionInProgressRepository;
   @Mock private RewardCalculatorConnector rewardCalculatorConnector;
-
+  @Mock private AuditUtilities auditUtilities;
```

Comment:
When it's a mock, let's explicitly it in the name (please could you fix also the other properties?)
```suggestion
  @Mock private AuditUtilities auditUtilitiesMock;
```